File: py/1318.py
from collections import defaultdict


def main() -> None:
    while True:
        n, m = [int(i) for i in input().split()]
        if n == 0:
            break
        tickets = [int(i) for i in input().split()]
        frequencies = defaultdict(int)
        for i in tickets:
            frequencies[i] += 1
        count = 0
        for i in frequencies:
            if frequencies[i] > 1:
                count += 1
        print(count)


if __name__ == '__main__':
    main()

File: cpp/1318.cpp
#include <iostream>
#include <unordered_map>
#include <vector>

using namespace std;

int main()
{
    unordered_map<int, int> frequencies;
    vector<int> tickets;
    int n, m, aux, count = 0;
    while (cin >> n >> m) {
        if (n == 0) {
            break;
        }
        for (int i = 0; i < m; i++) {
            cin >> aux;
            tickets.push_back(aux);
        }
        for (int i: tickets) {
            frequencies[i]++;
        }
        for (pair<const int, int> i: frequencies) {
            if (i.second > 1) {
                count++;
            }
        }
        cout << count << '\n';
        frequencies.clear();
        tickets.clear();
        count = 0;
    }
    return 0;
}